platforms/core-execution/worker-main/src/main/java/org/gradle/process/internal/worker/problem/WorkerProblemProtocol.java

```diff
@@ -26,4 +26,5 @@
  */
 public interface WorkerProblemProtocol {
     void reportProblem(Problem problem, @Nullable OperationIdentifier id);
+    void reportProblem(Problem problem, @Nullable OperationIdentifier id, String taskPath);
```

Comment:
:x: The `problem` is already there: why couldn't you encode the taskPath there, and not introduce a whole new parameter?

Reply:
Absolutely!